.github/workflows/build-wasm.yml

```diff
@@ -0,0 +1,38 @@
+name: WASM build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: "WebAssembly build ${{ github.event.ref }} for commit ${{ github.event.inputs.commit }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Output link to real commit
+        run: echo ${{ github.repository }}/commit/${{ github.event.inputs.commit }}
```

Comment:
Is this necessary? I picked that up from the windows build, we may not need it.

Reply:
It doesn't even look like doing anything...

Reply:
This one indeed wasn't necessary :heavy_check_mark: